Snippets/mongodb/MongoDB_3/MongoDB_3.csproj

```diff
@@ -4,7 +4,7 @@
     <LangVersion>7.3</LangVersion>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="NServiceBus.Storage.MongoDB" Version="3.0.0-alpha.225" />
-    <PackageReference Include="NServiceBus" Version="8.0.0-alpha.3024" /> <!--Remove when the MongoDB package is updated -->
+    <PackageReference Include="NServiceBus.Storage.MongoDB" Version="3.0.0-beta.1" />
+    <PackageReference Include="NServiceBus" Version="8.0.0-beta.1" /> <!--Remove when the MongoDB package is updated -->
```

Comment:
I saw these comments and I'm not sure if it's safe to remove these references. We can leave it for now. I don't see this being a big issue

Reply:
It is not just limited to this example, there are more. I think we can remove them all when we RC.